rke/k8s_rke_system_images.go

```diff
@@ -6394,6 +6394,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.6.3
```

Comment:
Hey @kinarashah, is `1.20.15-rancher1-2` going OOB? As per my understanding it was supposed to be going with v2.6.4.

Reply:
good catch @doflamingo721, thanks I will put in a separate pR to update!